python/runtime/cudaq/qis/py_qubit_qis.h

```diff
@@ -0,0 +1,18 @@
+/*************************************************************** -*- C++ -*- ***
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ *******************************************************************************/
```

Comment:
nit: 81 columns. (All of these were fixed mechanically at one point. Not sure why they came back.)